starkjs/fibonacci/input1.json

```diff
@@ -0,0 +1 @@
+[1,2]
```

Comment:
Can we make a data directory, and put this in it? or we can generate the input everytime randomly?

Reply:
I apply the data directory method because it will bring us stronger debugging capabilities